### PR TITLE
Fix eslint

### PR DESCRIPTION
**Pull request by zpoken**

### CODE REVIEW

File: packages/crypto/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,5 @@
   "extends": "tsconfig/base.json",
   "include": ["."],
   "exclude": ["dist", "build", "node_modules"],
-  "compilerOptions": {
-    "baseUrl": "."
-  }
+  "compilerOptions": {}
```
Comment (grod220) [inline packages/crypto/tsconfig.json:5]: Think you can remove `"compilerOptions": {}` entirely here

File: packages/eslint-config-custom/index.js [hunk 2]
```diff
@@ -13,6 +13,17 @@ module.exports = {
...
+      'warn',
+      {
+        whitelist: [
+          'bg\\-(card-radial|charcoal|teal|accent|background|teal/80|destructive|destructive/90)',
+          'text\\-(muted-foreground|accent-foreground|primary-foreground|destructive-foreground|primary|teal)',
+          'ring\\-(offset-background|ring)',
+          'border\\-(input|teal)',
+        ],
+      },
+    ],
```
Comment (grod220) [inline packages/eslint-config-custom/index.js:26]: Why is this necessary? This will be annoying to have to maintain.

Comment (grod220) [reply packages/eslint-config-custom/index.js:26]: When I remove this, I do not get any warning

Comment (grod220) [reply packages/eslint-config-custom/index.js:26]: By default, is `tailwindcss/no-custom-classname` off? When I remove these lines, I get no warnings. Are you adding in a new rule for extra protection?

Comment (zpoken) [reply packages/eslint-config-custom/index.js:26]: or do you not have a notification for this rule in your code? and a problem with my IDE
